ddtrace/contrib/kombu/util.py

```diff
@@ -0,0 +1,62 @@
+"""
+Some utils used by the dogtrace redis integration
+"""
+import kombu
+
+from ...ext import kombu as kombux, net
+
+PUBLISH_BODY_IDX = 0
+PUBLISH_ROUTING_KEY = 6
+PUBLISH_EXCHANGE_IDX = 9
+
+VALUE_PLACEHOLDER = "?"
+VALUE_MAX_LEN = 100
+VALUE_TOO_LONG_MARK = "..."
+CMD_MAX_LEN = 1000
+
+
+def _extract_conn_tags(connection):
+    """ Transform redis conn info into dogtrace metas """
+    try:
+        host, port = connection.host.split(':')
+        return {
+            net.TARGET_HOST: host,
+            net.TARGET_PORT: port,
+            kombux.VHOST: connection.virtual_host,
+        }
+    except Exception:
+        return {}
+
+
+def get_exchange_from_args(args):
+    """Extract the exchange"""
+    name = 'Unknown'
+
+    # Definition of _publish has changed, we're in trouble if != 11
+    if len(args) == 11:
```

Comment:
I don't think we need these checks. They don't guarantee us anything about the actual arguments. I think it suffices to ensure that the code is wrapped with a `try` and that we make sure to log any failures should they occur. 

This way if the version of `kombu` is not compatible with the integration then the user will be notified.

Reply:
Yeah. In general if a version is not compatible it's fine to avoid tracing, but users must be notified with logs so they can report an issue and we can easily fix that.